plugins/poll/assets/javascripts/discourse/components/poll-breakdown-option.gjs

```diff
@@ -48,4 +49,28 @@ export default class PollBreakdownOption extends Component {
 
     return htmlSafe(`background: ${color};`);
   }
+
+  <template>
+    <li
+      class="poll-breakdown-option"
+      style={{this.colorBackgroundStyle}}
+      {{on "mouseover" @onMouseOver}}
+      {{on "mouseout" @onMouseOut}}
+      role="button"
+    >
+      <span
+        class="poll-breakdown-option-color"
+        style={{this.colorPreviewStyle}}
+      ></span>
+
+      <span class="poll-breakdown-option-count">
+        {{#if this.showPercentage}}
+          {{this.percent}}%
```

Comment:
I know we haven't done this in the past, but we should start using a translation for percentages. Depending on the language, the percent sign either appears on the right or left side of numbers.

I suggest you add a new string in the `client.en.yml` locale file of Discourse (not the polls plugin). `js.number.percent` with a value of `"%{count}%"` seems appropriate.

```suggestion
          {{i18n "number.percent" count=this.percent}}
```

Reply:
done, incorporated